libs/npress/AdminModule/templates/@layout.latte

```diff
@@ -108,8 +109,8 @@
 			{block #TreeView}
 				{foreach $pages->getChildNodes() as $row}
 					<li id='menuid-{$row->id}'>
-						<div n:class='!$row[published] ? unpublished, $presenter->isCurrent($row->id) ? current'>
-							<a n:href='Pages:edit $row->id'>{$row->name}</a>
+						<div n:class='!$row[published] ? unpublished, $presenter->isCurrent($row->id) || $current === $row->id? current'>
```

Comment:
Dobrý pokus, ale mělo by se to dělat tak, že aktuální page dáš to `$template->page = PagesModel::getPageById($id_page)`. Je tam pak ošetřené, že to funguje správně i pro další případy (viz isCurrent metoda).